src/include/network/lan/address.rb

```diff
@@ -1350,8 +1355,6 @@ def AddressDialog
 
       Builtins.y2milestone("ShowAndRun: %1", ret)
 
-      LanItems.Rollback if ret == :abort
```

Comment:
So, why it is not needed now but was in the past? Was you trying to fix the removal of a new virtual interface when **editing**  it after has been added or it is a different scenario?.

Reply:
i'm not sure why it was needed before. However calling rollback is defined as abort callback and according to the logs it was called twice. So I dropped it.